src/Compilers/Core/Portable/Text/SourceText.cs

```diff
@@ -633,7 +633,19 @@ public virtual SourceText WithChanges(IEnumerable<TextChange> changes)
                 // there can be no overlapping changes
                 if (change.Span.Start < position)
                 {
-                    throw new ArgumentException(CodeAnalysisResources.ChangesMustBeOrderedAndNotOverlapping, nameof(changes));
```

Comment:
>ArgumentException [](start = 30, length = 17)

Are there no tests for the exceptional behavior?

Reply:
➡️ Tests are included for both normal and exceptional behavior

Reply:
Can you please point me to tests for the exceptional behavior? Are those existing tests?

---
In reply to: [187682548](https://github.com/dotnet/roslyn/pull/26798#discussion_r187682548) [](ancestors = 187682548)

Reply:
The only test I could find is one existing test `TestChangedTextWithMultipleOverlappingChanges`

---
In reply to: [188384892](https://github.com/dotnet/roslyn/pull/26798#discussion_r188384892) [](ancestors = 188384892,187682548)